src/lein_monolith/task/each.clj

```diff
@@ -86,7 +121,76 @@
           (if (get-in subproject [:profiles :monolith/inherited])
             [:default :monolith/inherited]
             [:default])))
-      (lein/apply-task (first task) subproject (rest task)))))
+      (config/debug-profile "apply-task"
+        (lein/resolve-and-apply subproject task)))))
+
+
+(defn- run-task!
+  "Runs the given task, returning a map of information about the run."
+  [ctx i target]
+  ; Try to reclaim some memory before running the task.
+  (System/gc)
+  (let [start (System/nanoTime)
+        opts (:opts ctx)
+        subproject (get-in ctx [:subprojects target])
+        results (delay {:name target
+                        :index i
+                        :elapsed (/ (- (System/nanoTime) start) 1000000.0)})]
+    (try
+      (lein/info (format "\nApplying to %s (%s/%s)"
+                         (ansi/sgr target :bold :yellow)
+                         (ansi/sgr (inc i) :cyan)
+                         (ansi/sgr (:num-targets ctx) :cyan)))
+      (apply-subproject-task (:monolith ctx) subproject (:task ctx))
+      (assoc @results :success true)
+      (catch Exception ex
+        (when-not (or (:parallel opts) (:endure opts))
```

Comment:
You could potentially support the resume command in parallel mode, which would eliminate all dependencies of the given subproject from the list of subprojects that need to be run. Kind of like a "negative :subtree".